### PR TITLE
Fix small issues

### PR DESCRIPTION
**Pull request by jsteemann**
The individual commits in this PR should be self-explanatory.
All small and _very_ low-priority changes.

### CODE REVIEW

File: include/rocksdb/status.h [hunk 1]
```diff
@@ -307,14 +307,11 @@ class Status {
...
  private:
-  // A nullptr state_ (which is always the case for OK) means the message
-  // is empty.
-  // of the following form:
-  //    state_[0..3] == length of message
-  //    state_[4..]  == message
   Code code_;
   SubCode subcode_;
   Severity sev_;
+  // A nullptr state_ (which is always the case for OK) means the message
```
Comment (yiwu-arbug) [inline include/rocksdb/status.h]: Seems empty message is also used for NotFound status, e.g. https://github.com/facebook/rocksdb/blob/master/db/version_set.cc#L1800

Comment (jsteemann) [inline include/rocksdb/status.h]:
```suggestion
  // A nullptr state_ (which is at least the case for OK) means the extra message
```

**Comment by jsteemann** [comment]
@yiwu-arbug : looks like the Ci tests succeeded except one of the builds on TravisCI, which hit the build timeout of 50 minutes. 
Is there any chance to get this PR merged? Thanks!

**Comment by yiwu-arbug** [comment]
@jsteemann sorry, I'm not on the team any more. Maybe someone else could help.

**Pull request closed without merging**